sspl_test/run_qa_test.sh

```diff
@@ -152,6 +156,17 @@ restore_cfg_services()
         conf "$global_config" set "server_node>$machine_id>site_id=DC01"
         conf "$global_config" set "server_node>$machine_id>rack_id=RC01"
         conf "$global_config" set "server_node>$machine_id>cluster_id=CC01"
+        if [ "${CVG_NAME}" == "null" ]
```

Comment:
Where CVG_NAME is defined ?

Reply:
I have updated this variable value in line no. 248.
By default, any undefine variable value is null only.